src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/validation/singleField/singleFieldValidationSagas.ts

```diff
@@ -1,35 +1,41 @@
-import { AxiosRequestConfig } from 'axios';
 import { SagaIterator } from 'redux-saga';
 import { call, select, takeLatest } from 'redux-saga/effects';
+import { AxiosRequestConfig } from 'axios';
 import { IRuntimeStore } from 'src/types';
-import { convertDataBindingToModel } from '../../../../utils/databindings';
-import { put } from '../../../../utils/networking';
-import { mapApiValidationsToRedux } from '../../../../utils/validation';
+import { IRuntimeState } from 'src/types';
+import { get } from '../../../../utils/networking';
+import { mapDataElementValidationToRedux } from '../../../../utils/validation';
 import Actions from '../validationActions';
 import { IRunSingleFieldValidationAction } from './singleFieldValidationActions';
 import * as ActionTypes from '../validationActionTypes';
+import { getValidationUrl } from 'src/utils/urlHelper';
+import { ILayoutState } from '../../layout/formLayoutReducer';
+import FormValidationActions from '../validationActions';
 
 export function* runSingleFieldValidationSaga({
-  url,
   dataModelBinding,
 }: IRunSingleFieldValidationAction): SagaIterator {
-  const state: IRuntimeStore = yield select();
-  try {
-    const requestBody = convertDataBindingToModel(state.formData.formData);
-    const config: AxiosRequestConfig = {
+    const state: IRuntimeState = yield select();
+    const LayoutSelector: (store: IRuntimeStore) => ILayoutState = (store: IRuntimeStore) => store.formLayout;
+    const layoutState: ILayoutState = yield select(LayoutSelector);
```

Comment:
You don't need lines 19-20, you can just use the overall `state`-object. This also contains the layout state (`state.formLayout.layouts`)